index.js

```diff
@@ -0,0 +1,123 @@
+// Import stylesheets
+import './style.scss';
+
+var redDisplay = document.querySelector(".red");
+var greenDisplay = document.querySelector(".green");
+var blueDisplay = document.querySelector(".blue");
+var easyButton = document.querySelector(".easy");
+var hardButton = document.querySelector(".hard");
+var newColors = document.querySelector(".new-game");
+var squaresList = document.querySelectorAll("li.square");
+var modeButtons = document.querySelectorAll(".mode");
+
+var BG_COLOR = "#4e82b4";
+var isEasyGame = false;
+var isGameOver = false;
+
+function startGame() {
+  isGameOver = false;
+  document.querySelector("header").style.backgroundColor = BG_COLOR;
+  var num = isEasyGame ? 3 : 6;
+  var colors = getRandomColors(num);
+  for(var i = 0; i < num; i++){
+      var color = colors[i];
+      squaresList[i].style.backgroundColor = rgbToHex(color.red, color.green, color.blue);
+  }
+  //select the correct guess
+  var index = getRandomValue(0,num - 1); 
+  //set the header RGB display
+  redDisplay.textContent = colors[index].red;
+  greenDisplay.textContent = colors[index].green;
+  blueDisplay.textContent = colors[index].blue;
+}
+
+function componentToHex(c) {
```

Comment:
For better readability, let's move all those generic function to some `utils.js` file.

Reply:
decided not to split the JS files eventually